File: lib_internet/src/main/java/world/share/lib_internet/http/exception/NetErrorBean.java
package world.share.lib_internet.http.exception;

/**
 * @author mac
 */
public class NetErrorBean {
    private int code;

    private String message;

    public NetErrorBean(int code, String message) {
        this.code = code;
        this.message = message;
    }

    public int getCode() {
        return code;
    }

    public void setCode(int code) {
        this.code = code;
    }

    public String getMessage() {
        return message == null ? "" : message;
    }

    public void setMessage(String message) {
        this.message = message;
    }

    @Override
    public String toString() {
        return "NetErrorBean{" +
                "code=" + code +
                ", message='" + message + '\'' +
                '}';
    }

}

File: lib_base/src/main/java/world/share/lib_base/internet/http/exception/NetErrorBean.kt
package world.share.lib_base.internet.http.exception

/**
 * @author mac
 */
class NetErrorBean(var code: Int, private var message: String?) {
    fun getMessage(): String {
        return if (message == null) "" else message!!
    }

    fun setMessage(message: String?) {
        this.message = message
    }

    override fun toString(): String {
        return "NetErrorBean{" +
                "code=" + code +
                ", message='" + message + '\'' +
                '}'
    }
}